examples/keygen/keygen.c

```diff
@@ -49,13 +51,15 @@ int TPM2_Keygen_Example(void* userCtx, int argc, char *argv[])
     int rc;
     WOLFTPM2_DEV dev;
     WOLFTPM2_KEY storage; /* SRK */
+    WOLFTPM2_KEY aesKey; /* Symmetric key */
     WOLFTPM2_KEYBLOB newKey;
     TPMT_PUBLIC publicTemplate;
-    TPMI_ALG_PUBLIC alg = TPM_ALG_RSA; /* TPM_ALG_ECC */
+    TPMI_ALG_PUBLIC alg = TPM_ALG_RSA; /* default, see usage() for options */
     TPM_ALG_ID paramEncAlg = TPM_ALG_NULL;
     WOLFTPM2_SESSION tpmSession;
     TPM2B_AUTH auth;
     int bAIK = 1;
+    int keyBits = 256;
```

Comment:
This will always create an AES key with 256-bit?

Reply:
yes, indeed. I will address this by adding option for the keysize, only when symmetric key is chosen. This might complicate the arguments handling further.